bin/zkServer.sh

```diff
@@ -151,6 +151,8 @@ fi
 
 ZOO_LOG_FILE=${ZOO_LOG_FILE:-zookeeper-$USER-server-$HOSTNAME.log}
 _ZOO_DAEMON_OUT="$ZOO_LOG_DIR/zookeeper-$USER-server-$HOSTNAME.out"
+ZOO_LOG_FORMAT=${ZOO_LOG_FORMAT:-pattern}
+ZOO_VERSION=$($JAVA -cp "$CLASSPATH" org.apache.zookeeper.version.VersionInfoShortMain)
```

Comment:
I don't think that this is a good idea, why  do you need it ?
you are bootstrapping a JVM here

Reply:
I'm aware of that, it was a hack since I could not find another var. However I'll try to initialize `zookeeper.version` statically instead.